modules/illumina.nf

```diff
@@ -11,9 +11,22 @@ process performHostFilter {
 
     output:
     tuple val(sampleName), path("${sampleName}_hostfiltered_R1.fastq.gz"), path("${sampleName}_hostfiltered_R2.fastq.gz"), emit: fastqPairs
+    tuple val(sampleName), path("${sampleName}_performHostFilter_provenance.yml"), emit: provenance
 
     script:
     """
+    printf -- "- process_name: performHostFilter\\n"                                           >> ${sampleName}_performHostFilter_provenance.yml
+    printf -- "  tools:\\n"                                                                    >> ${sampleName}_performHostFilter_provenance.yml
+    printf -- "    - tool_name: bwa mem\\n"                                                    >> ${sampleName}_performHostFilter_provenance.yml
```

Comment:
Could you adjust to:

```yml
- tool_name: bwa
  subcommand: mem
```

...and:

```yml
- tool_name: samtools
  subcommand: sort
```

See this example on the plasmid-screen pipeline:

https://github.com/BCCDC-PHL/plasmid-screen/blob/2898065e1702c00d5b6acb38565f79ac81cd0633/modules/align_reads_to_reference_plasmid.nf#L24-L39